supervisor/supervisor.go

```diff
@@ -333,8 +333,10 @@ func (s *Supervisor) restore() error {
 			continue
 		}
 		processes, err := container.Processes()
```

Comment:
keep this error check here will do no harm, but I am happy to remove it because it's not in use. not sure if this error returned is a planned design.